samples/EventHub/allowed-event-hub-firewall-ip/README.md

```diff
@@ -0,0 +1,26 @@
+# Allowed only selected IPs for Event hub firewall
+
+This policy allows user to specify selected IPs for event hub firewall from specified array of IPs. You specify an array of predefined IPs for event hub firewall. 
```

Comment:
>This policy allows user t [](start = 0, length = 25)

please make this more clear that the policy only restricts what IPs can be added to the firewall, it doesn't automatically add them to the firewall (nor does it require ALL of them to be there)

Reply:
changed to - This policy will restrict adding firewall IPs in event hub and allow only those IPs which are predefined in array of selected IPs.